src/thruster.cpp

```diff
@@ -272,11 +276,11 @@ void Thruster::Update()
     // that would enable you to disable any auto disabling. Although messages
     // are still being sent even with visualizers off, I don't think this is in
     // dire need of optimization right now.
-    // This is currently being run at a hardcoded 5hz.
-    if(ros::Time::now() - last_update_time > visualize_update_time)
+    if(ros::Time::now() - last_update_time > visualizer_update_time)
     {
-        UpdateVisualizers();
         last_update_time = ros::Time::now();
+        UpdateVisualizers();
+        ReloadParams();
```

Comment:
Because then they are only reloaded, like, 10 times a second rather than every frame. It's just the best place for the call.